### PR TITLE
add options to pull other parameters from ini file

### PR DESCRIPTION
**Pull request by fuzzy-logic-zach**
## Pull Request Checklist

**Is this in reference to an existing issue?**
Yes, issue #85 

#### General

- [√] Update Changelog following the conventions laid out [here](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md)

- [ ] Update README with any necessary configuration snippets

- [ ] Binstubs are created if needed

- [ ] RuboCop passes

- [ ] Existing tests pass

#### New Plugins

- [ ] Tests

- [ ] Add the plugin to the README

- [ ] Does it have a complete header as outlined [here](http://sensu-plugins.io/docs/developer_guidelines.html#coding-style)

### CODE REVIEW

**Comment by majormoses** [comment]
Thanks for your contribution to Sensu plugins! Without people like you submitting PRs we couldn't run the project. I will review it shortly.

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format listed [here](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md)
 
 ## [Unreleased]
+### Fixed
```
Comment (majormoses) [inline CHANGELOG.md]: This would be added, its not a bug but an enhancement.

File: bin/check-mysql-select-count.rb [hunk 2]
```diff
@@ -85,11 +84,13 @@ def run
...
       db_user = section['user']
       db_pass = section['password']
+      config[:host] = section['host']
+      config[:socket] = section['socket']
     else
       db_user = config[:username]
       db_pass = config[:password]
     end
-    raise "invalid query : #{config[:query]}" unless config[:query] =~ /^select\s+count\(\s*\*\s*\)/i
+    raise 'Please specify hostname using -h or in mysql.cnf file' unless config[:host]
```
Comment (majormoses) [inline bin/check-mysql-select-count.rb]: If you want add more exception handling that's fine but why did you remove the existing one?

**Comment by majormoses** [comment]
I fixed up CI for you

File: bin/check-mysql-select-count.rb (hunk 2 above)
Comment (law) [reply bin/check-mysql-select-count.rb]: I think I can speak to this, I helped Zach a very little bit on this PR and he is indisposed at the moment.  With line 92 in place, a query like: 
`select COUNT from INFORMATION_SCHEMA.INNODB_METRICS where NAME = 'trx_rseg_history_len'\G`

would fail, because it's not specifically a "select count(*)..."-type query, even though it's only returning a single row.  Certainly open to more graceful ways of handling the error-checking, but this way seemed the most expedient without further context of why 'select count(*)...'-type queries were originally the only ones allowed.

Comment (majormoses) [reply bin/check-mysql-select-count.rb]: I see and that makes sense however this increases the ease of a bad SQL statement which looking at the code was already quite easy. However this is not very easy to do with the current library so I think we are just going to need to assume trust that your sensu stack is properly secured (most are not) and assume that your CM tool does not get compromised and can push out a malicious config. I am gonna hold off for another day and see if I can come up with something other than re-writing this to use use something like active record.

Comment (majormoses) [reply bin/check-mysql-select-count.rb]: I thought more about it and there is no real easy solution available so I am gonna merge this.

File: CHANGELOG.md (hunk 1 above)
Comment (majormoses) [inline CHANGELOG.md]:
```suggestion
### Added
```

**Comment by majormoses** [comment]
released: https://rubygems.org/gems/sensu-plugins-mysql/versions/2.6.0
